src/hotspot/share/opto/addnode.hpp

```diff
@@ -42,6 +42,15 @@ typedef const Pair<Node*, jint> ConstAddOperands;
 // by virtual functions.
 class AddNode : public Node {
   virtual uint hash() const;
+
+private:
```

Comment:
Can be removed since these methods are already private by default here since it's a `class` and not a `struct`.
```suggestion
```